.circleci/config.yml

```diff
@@ -2312,6 +2337,9 @@ workflows:
       - pytorch_cpp_doc_push:
           requires:
             - pytorch_linux_xenial_py3_6_gcc5_4_build
+      - pytorch_doc_test:
+          requires:
+            - pytorch_linux_xenial_py3_6_gcc5_4_build
```

Comment:
Do we really need to wait for the binary build to complete to run doc tests?

Reply:
we do need a working version of PyTorch, because the doc tests will evaluate the python code in the docs to make sure they work